Joedmin-hw3/Joedmin-hw3/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace Joedmin_hw3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.Write("What do you want to do? login/register/exit: ");
+                string answer = Console.ReadLine();
+
+                // Exit
+                if (String.Equals(answer, "exit", StringComparison.OrdinalIgnoreCase))
+                {
+                    Process.GetCurrentProcess().Kill();
+                }
+
+                // Login
+                else if (String.Equals(answer, "login", StringComparison.OrdinalIgnoreCase))
+                {
+                    using (var stream = new StreamReader("Users.txt"))
+                    {
+                        Dictionary<string, string> usersAndPasswords = GetCredentials("Users.txt");
+
+                        string[] credentials = GetCredencials();
+
+                        if (usersAndPasswords.ContainsKey(credentials[0]))
```

Comment:
This will be a case-sensitive username comparison.  There's a way you can use a dictionary and still have a case insensitive username comparison.

Reply:
I know, this is how I meant to do it, they can differ by that, but if that's wrong, I'm able to change it